huaweicloud/services/acceptance/identitycenter/data_source_huaweicloud_identitycenter_permission_set_provisioning_accounts_test.go

```diff
@@ -0,0 +1,65 @@
+package identitycenter
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourcePermissionSetProvisioningAccounts_basic(t *testing.T) {
+	dataSource := "data.huaweicloud_identitycenter_permission_set_provisioning_accounts.test"
+	rName := acceptance.RandomAccResourceName()
+	dc := acceptance.InitDataSourceCheck(dataSource)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
```

Comment:
suggest to add precheck for environment variable used by `testProvisionPermissionSet_basic(name)`